Lib/test/test_py_compile.py

```diff
@@ -192,6 +192,14 @@ def test_invalidation_mode(self):
                 fp.read(), 'test', {})
         self.assertEqual(flags, 0b1)
 
+    def test_quiet(self):
+        bad_coding = os.path.join(os.path.dirname(__file__), 'bad_coding2.py')
+        with support.captured_stderr():
```

Comment:
```suggestion
        with support.captured_stderr() as stderr:
```

Then we can add something like

```py
self.assertEqual(stderr.getvalue(), '')
```

to test the `quiet` option.

Reply:
Why did you mark this resolved? It still needs to be addressed.

Reply:
Sorry, I had confused it with @vadmium 's comment .